.github/workflows/release.yml

```diff
@@ -90,14 +90,49 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_API_KEY }}
           BUDIBASE_RELEASE_VERSION: ${{ steps.previoustag.outputs.tag }}
 
+  release-helm-chart:
+    needs: [release-images]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Helm
+        uses: azure/setup-helm@v1
+        id: helm-install
+
+      # due to helm repo index issue: https://github.com/helm/helm/issues/7363
+      # we need to create new package in a different dir, merge the index and move the package back
+      - name: Build and release helm chart
+        run: |
+          git config user.name "Budibase Helm Bot"
+          git config user.email "<>"
+          git reset --hard
+          git pull
+          mkdir sync
+          echo "Packaging chart to sync dir"
+          helm package charts/budibase --version 0.0.0-master --app-version "$RELEASE_VERSION" --destination sync
+          echo "Packaging successful"
+          git checkout gh-pages
+          echo "Indexing helm repo"         
+          helm repo index --merge docs/index.yaml sync
+          mv -f sync/* docs
+          rm -rf sync
+          echo "Pushing new helm release"
+          git add -A
+          git commit -m "Helm Release: ${{ env.RELEASE_VERSION }}"
+          git push
+        env:
+          RELEASE_VERSION: ${{ steps.previoustag.outputs.tag }}
+
+
   # Trigger deploy to new EKS preprod environment
   trigger-deploy-to-preprod-env:
     needs: [release-helm-chart]
     runs-on: ubuntu-latest
     steps:
       - uses: passeidireto/trigger-external-workflow-action@main
         env:
-          PAYLOAD_VERSION: ${{ env.RELEASE_VERSION }}
+          PAYLOAD_VERSION: ${{ steps.previoustag.outputs.tag }}
```

Comment:
in the deploys repo we are looking for:
```
VERSION: ${{ github.event.client_payload.VERSION }}
```
So should this line read:
```
VERSION: ${{ steps.previoustag.outputs.tag }}
```
Or we change in the other repo

Reply:
the `PAYLOAD_` prefix is required by the `trigger-external-workflow-actio` action. It uses that identifier to add to the payload of the request